lib/about.dart

```diff
@@ -1,76 +1,74 @@
+  @override
+  _AboutState createState() => _AboutState();
+}
+
+class _AboutState extends State<About> {
+
+
+  List<String> names = [];
+  List<String> urls = [];
+  bool isLoading = true;
+
+
 
-class About extends StatelessWidget {
   @override
-  Widget build(BuildContext context){
+  void initState() {
+    super.initState();
+    getUsername();
+  }
+
+
+  void getUsername() async {
+    List<String> nameList = [];
+    List<String> urlList = [];
+    Response response = await get(
+        'https://api.github.com/repos/enzaimz/c_made_easy/contributors');
+    final result = jsonDecode(response.body);
+
+    for (var map in result) {
+      nameList.add(map['login']);
+      urlList.add(map['html_url']);
+    }
+
+    setState(() {
+      isLoading = false;
+      names = nameList;
+      urls = urlList;
+    });
+  }
+
+  
+  @override
+  Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
-        title: Text("About"),
-      ),
-      body: ListView(
-        children:<Widget>[
-          SizedBox(
-            height: 15.0,
-          ),
-          Center(
-            child:Text(
-              "Developed by:",
-              style: new TextStyle(
-                fontSize: 30.0,
-              ),
-            ),
-          ),
-          SizedBox(
-            height: 10.0,
-          ),
-            
-          Card(
-            child:new Container(
-              padding: EdgeInsets.all(20.0),
-              child: Link(
-                child: new Text('Hariom Verma',
-                style: new TextStyle(
-                  fontSize: 23.0,
-                ),
-                ),
-                url: 'https://github.com/harry-hov',                
-              ),
-            ), 
-          ),
-          
-          SizedBox(
-            height: 10.0,
-          ),
-          
-          Card(
-            child:new Container(
-              padding: EdgeInsets.all(20.0),
-              child: Link(
-                child: new Text('Uday Mewada',
-                style: new TextStyle(
-                  fontSize: 23.0,
-                ),
-                ),
-                url: 'https://github.com/udaymewada',                
-              ),
-            ), 
-          ), 
-          SizedBox(
-            height: 20.0,
-          ),
-          Center(
-            child:Text(
-              "© enzaimz",
-              
-              style: new TextStyle(
-                fontSize: 20.0,
-                fontWeight: FontWeight.w200,
-              ),
-            ),
-          ),
-        ],
+        title: Text('About Developers'),
       ),
+     body: isLoading
+          ? Center(child: CircularProgressIndicator())
+          : ListView.builder(
+              itemCount: names.length,
+              itemBuilder: (context, index) {
+                return Card(
+                  child: Container(
+                    padding: EdgeInsets.all(20.0),
+                    child: Link(
```

Comment:
Can we ListTile inside card?
Something like:
```
ListTile(
    title: Text(Name),
    subtitle: Text('GitHub: ' + username),
),
```

Reply:
i put the list tile 

![Screenshot_1607418815](https://user-images.githubusercontent.com/56194329/101463957-16fb1a80-3964-11eb-9d3d-1a73f6f1b10d.png)

Reply:
It looks good. Can you also fetch name?